Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -515,7 +515,7 @@ public static void Update()
             foreach (KeyValuePair<uint, NetworkIdentity> kvp in NetworkIdentity.spawned)
             {
                 NetworkIdentity identity = kvp.Value;
-                if (identity != null)
+                if ((object)identity != null)
```

Comment:
We need this check here incase identity is destroyed. We should never be adding real null to spawned